Resources/config/phpcr-sitemap.xml

```diff
@@ -15,5 +15,11 @@
 
             <tag name="cmf_seo.sitemap.loader" priority="-2"/>
         </service>
+        <service id="cmf_seo.sitemap.phpcr.depth_guesser" class="Symfony\Cmf\Bundle\SeoBundle\Sitemap\DepthGuesser">
+            <argument type="service" id="doctrine_phpcr"/>
+            <argument>%cmf_routing.dynamic.persistence.phpcr.content_basepath%</argument>
+
+            <tag name="cmf_seo.sitemap.guesser" position="-2"/>
```

Comment:
but the other guessers are at -2 as well. which maybe is not a great
choice but changing that would be a BC break. i think its less confusing
to keep this in line with the others.

people that need to extend can always go to positive or higher negative
numbers. from the cmf side, nothing is at the default priority.